src/test/java/org/assertj/core/error/ShouldBeEqualByComparingFieldByFieldRecursively_create_Test.java

```diff
@@ -66,8 +66,8 @@ public void should_use_unambiguous_fields_description_when_standard_description_
     List<Difference> differences = determineDifferences(withHashSet, withSortedSet, null, null);
     // WHEN
     // @format:off
-    String message = shouldBeEqualByComparingFieldByFieldRecursive(withSortedSet,
-                                                                   withHashSet,
+    String message = shouldBeEqualByComparingFieldByFieldRecursive(withHashSet,
```

Comment:
I believe this test and the one below were giving false positives: actual and expected are inverted, but it would still pass since they contain the same elements.